tests/strictEqual.test.js

```diff
@@ -0,0 +1,33 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictEqual')
+
+
+describe('strictEqual', () => {
+
+  it('returns false if data types are not equal', () => {
+    const checkOne = 5
+    const checkTwo = '5'
+    const checkResult = strictEqual(checkOne, checkTwo)
```

Comment:
This data is simple enough that you could one line this and not lose any context

```JavaScript
const checkResult = strictEqual(5, '5')
```